.golangci.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+linters:
+  disable:
+    - errcheck      # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - mnd
+    - paralleltest
+    - staticcheck   # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - testpackage
+    - unused
+  exclusions:
+    generated: lax
```

Comment:
lax is the default.  Should omit the default values? Alternatively, is there any reason not to disable this exclusion?

Reply:
I've dropped the defaults and exclusions.  We can add them later if they end up actually triggering something.

Reply:
I'm not seeing an update.   Hopefully my changes aren't causing you merge problems. 😬

Reply:
Finished pushing up my edits

Reply:
(i keep thinking it's like g4 where i batch up my replies and send when done rather than each comment sends when it gets added)